DNN Platform/Library/Services/FileSystem/IFileInfo.cs

```diff
@@ -39,7 +39,7 @@ public interface IFileInfo
         int PortalId { get; set; }
         string RelativePath { get; }
         string SHA1Hash { get; set; }
-        int Size { get; set; }
+        long Size { get; set; }
```

Comment:
This creates backward compatibility issue. Must deprecate and create another one.

Reply:
George, agreed, if a 3rd party module tries to copy into an int variable.
but the only alternative would be a n ew property LongSize with true value and keeping (deprecated) size property, set to -2, if size exceed max value. I am not sure, whether this is a better solution